### PR TITLE
ja: Make docs/reference/glossary/kube-apiserver.md follow v1.18 of the original text 

### LINKED ISSUE #25215: ja: Make docs/reference/glossary/kube-apiserver.md follow v1.18 of the original text
**This is a Feature Request**

**What would you like to be added**
Update docs/reference/glossary/kube-apiserver.md to follow v1.18 of the original (English) text.

**Why is this needed**
content/ja/docs/reference/glossary/kube-apiserver.md is outdated.

**Comments**
/language ja
/good-first-issue

File to update
https://github.com/kubernetes/website/blob/dev-1.18-ja.2/content/ja/docs/reference/glossary/kube-apiserver.md

Original
https://github.com/kubernetes/website/blob/fb6364d/content/en/docs/reference/glossary/kube-apiserver.md

diff between translated and v1.18
https://gist.github.com/a4deffae4dfcd79f2281ca337b671096

**Note**
Currently, we use **dev-1.18-ja.2** branch. Please open a PR targeting the branch.

ref
https://kubernetes.io/ja/docs/contribute/localization/

If you have a question, feel free to ask us at slack.k8s.io #kubernetes-docs-ja channel.

### PR DESCRIPTION
**Pull request by KobayashiD27**
Update "full link" in `reference/glossary/kube-apiserver.md`

Fix #25215

### CODE REVIEW

File: content/ja/docs/reference/glossary/kube-apiserver.md [hunk 1]
```diff
@@ -19,4 +19,4 @@ tags:
 
 Kubernetes APIサーバーの主な実装は[kube-apiserver](/docs/reference/generated/kube-apiserver/)です。
 kube-apiserverは水平方向にスケールするように設計されています&mdash;つまり、インスタンスを追加することでスケールが可能です。
-複数のkube-apiserverインスタンスを実行することで、インスタンス間でトラフィックを分散させることが可能です。
\ No newline at end of file
+複数のkube-apiserverインスタンスを実行することで、インスタンス間でトラフィックを分散させることが可能です。
```
Comment (inductor) [inline content/ja/docs/reference/glossary/kube-apiserver.md:22]: not a big deal but can you tell what is the diff here?

Comment (KobayashiD27) [reply content/ja/docs/reference/glossary/kube-apiserver.md:22]: Sorry, I don't know why the diff is here...
I think I didn't touch this line.

Comment (inductor) [reply content/ja/docs/reference/glossary/kube-apiserver.md:22]: Please undone the diff if possible. If it is necessary that is okay but let me know.

Comment (KobayashiD27) [reply content/ja/docs/reference/glossary/kube-apiserver.md]: I have been able to remove this diff.

**Pull request merged**